### ISSUE TITLE
User experience on installing WSO2 APIM (common to all products) with CA-signed certificates

### ISSUE THREAD

**Issue by Mariangela** [wso2/docs-apim#3106]
**Description:**
Chanaka Fernando reported as follows:

A user has failed at the point when accessing the portal URLs after starting the server. Some organizations may have certain policies and browsers that do not allow passing through this certificate issue (adding an exception). In such a scenario, the user is stuck and our documentation does not seem to guide the user in the right path.

Usually, installation should work without any issue. But when the user tries to run the product, we have the following documentation.

https://apim.docs.wso2.com/en/latest/install-and-setup/install/running-the-product/

In this document, there is no direct link to the certificate installation page. That means if the user is stuck with the certificate error, he cannot get over this hurdle and nothing can be done. We have the below documents that explain how to install a valid certificate.

https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/configuring-keystores/configuring-keystores-in-wso2-api-manager/
https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/configuring-keystores/keystore-basics/creating-new-keystores/

Shall we fix this documentation with proper links and make sure that users are properly guided through the steps? I think we should give more prominence to this certificate installation step rather than just mentioned that as "Info" section. Some users might think that our product is not properly when they see that certificate error as the first thing when they try to access the portal. If possible, we should put a valid certificate and get rid of this error entirely. 

The bottom line is that users are getting confused about this behavior and our documentation does not cover the reasoning behind this behavior and guide the user in the correct direction. This is a P1 in terms of user experience and better fix it soon.
